### DISCUSSION TITLE
[Export Data] Add two buttons to Skip/Export All columns

### DISCUSSION THREAD

**Discussion by PsyLogic** [dbeaver/dbeaver#18768] [Ideas]
Hello,

Recently I tried to export a table that has over 30 columns and I tried to skip the majority of them but it was a pain to do so.
I hope if there's two buttons "**Skip All**", "**Export All**"  (or maybe a checkbox with the same idea "**Select/Unselecet All**").

Thank you

![image](https://user-images.githubusercontent.com/8070135/212367550-34ba345d-8d18-40c4-978e-a6745646bbf5.png)

**Comment by ShadelessFox** [comment]
Hello @PsyLogic,

I've created a FR based on this discussion: #18774
